test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -212,6 +225,26 @@ contract('CoreIssuanceOrder', accounts => {
         zeroExOrderTakerAssetAmount,                      // amount of zeroExOrder to fill
       );
 
+      // Create Kyber trade for the third component, using ether(25) makerToken. Conversion rate pre set on snapshot
+      const sourceTokenQuantity = ether(25);
+      const maxDestinationQuantity = issuanceOrderRequiredComponentAmounts[2];
+      const componentTokenDecimals = (await thirdComponent.decimals.callAsync()).toNumber();
+      const sourceTokenDecimals = (await makerToken.decimals.callAsync()).toNumber();
+      kyberConversionRatePower = new BigNumber(10).pow(18 + sourceTokenDecimals - componentTokenDecimals);
+      const minimumConversionRate = maxDestinationQuantity.div(sourceTokenQuantity)
+                                                          .mul(kyberConversionRatePower)
```

Comment:
I love the math steps put on their own line

Reply:
+1